repos/system_upgrade/el8toel9/actors/checkarmbootloader/actor.py

```diff
@@ -0,0 +1,22 @@
+import leapp.libraries.actor.checkarmbootloader as checkarmbootloader
+from leapp.actors import Actor
+from leapp.reporting import Report
+from leapp.tags import ChecksPhaseTag, IPUWorkflowTag
+
+
+class CheckArmBootloader(Actor):
+    """
+    Inhibit ARM system upgrades on path with incompatible kernel/bootloader
+
+    Due to an incompatibility of RHEL8 bootloader with newer versions of kernel
+    on RHEL9 since version 9.5 the upgrade cannot be performed as the old
+    bootloader is used to load the new versions of kernel during the upgrade.
```

Comment:
Yes, but this kinda does not describes the issue. The old loader is always used to load the new kernel. Maybe rephrase it to something like `the old bootloader cannot load the new kernel`.

Reply:
Sure. Rephrased!